association.go

```diff
@@ -406,6 +405,10 @@ func (a *Association) readLoop() {
 	}
 
 	a.log.Debugf("[%s] readLoop exited", a.name)
+
+	// also stop writeLoop, otherwise writeLoop can be leaked
+	// if connection is lost when there is no writing packet.
+	a.closeWriteLoopOnce.Do(func() { close(a.closeWriteLoopCh) })
```

Comment:
@at-wat Here's my small request...  Can you move this line into the defer above, where maybe below the `close(a.readLoopCloseCh)`?  (just to put similar lines together)

Reply:
Sure, it sounds better.